docs/API/README.md

```diff
@@ -1,6 +1,9 @@
 # API Mock Tests
 The [Swagger](swagger.yaml) describes the API used by the Integration repository to support the `catalog`, `registry` and `store` functionalities.
 
+Its possible to visualize the REST API using any on the numerous [online editors](https://editor.swagger.io/)
```

Comment:
How does this work when our swagger file references relative files?

Reply:
+1 - this won't work if it cant access the relative path

Reply:
yes - though this is just a reminder on how to visualize this since most IDE have internal capabilities for showing this within ...